File: Python/Uri-Judge/AD HOC/Iccanobif.py
n=int(input())
lista=[]

for i in range(n):
    if(i==0 or i==1):
        lista.append(1)
    else:
        lista.append(lista[i-1]+lista[i-2])
lista2=lista[::-1]
resultado = ' '.join(map(str, lista2))
print(resultado)
    
   

File: dart/Iccanobif.dart
import 'dart:io';

void main(List<String> args) {
  int a = int.parse(stdin.readLineSync()!);
  var lista = [];

  for (int i = 0; i < a; i++) {
    if (i == 0 || i == 1) {
      lista.add(1);
    } else {
      lista.add(lista[i - 1] + lista[i - 2]);
    }
  }

  var resultado = lista.reversed.toList().join(" ");

  print(resultado);
}
